exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -114,7 +129,7 @@ def __init__(
         self,
         endpoint: str = "localhost:55680",
         credentials: ChannelCredentials = None,
-        metadata: tuple = None,
+        metadata: Tuple[Any] = None,
```

Comment:
https://grpc.io/docs/what-is-grpc/core-concepts/#metadata

Here's an example I found in the official repo https://github.com/grpc/grpc/blob/eaee09913f39c48f072eab55240e0de35567b610/examples/python/metadata/metadata_client.py#L33-L38